linux_os/guide/system/accounts/accounts-restrictions/root_logins/use_pam_wheel_for_su/rule.yml

```diff
@@ -24,6 +24,7 @@ identifiers:
 
 references:
     cis@rhel7: "5.7"
+    cis@rhel8: 5.7
```

Comment:
I am sorry, but you have to quote the `5.7` for the time being, because otherwise it is interpreted as a number, which confuses the build system.

Reply:
@matejak Ah that makes sense! I guess the references with multiple `.` delimiters are treated as strings automatically?